src/protagonist/API.Tests/Integration/DefaultDeliveryChannelsTests.cs

```diff
@@ -509,4 +506,212 @@ public async Task Delete_DeleteANonExistentDefaultDeliveryChannelForCustomer_404
+            d.Name == "default-audio");
+        
+        var dbEntry = dlcsContext.DefaultDeliveryChannels.Add(new DLCS.Model.DeliveryChannels.DefaultDeliveryChannel()
+        {
+            Customer = customerId,
+            MediaType = mediaType,
+            DeliveryChannelPolicyId = deliveryChannelPolicy.Id,
+            Space = space
+        });
+        await dlcsContext.SaveChangesAsync();
+
+        var path = $"customers/{customerId}/spaces/{space}/defaultDeliveryChannels/{dbEntry.Entity.Id}";
+
+        // Act
+        var response = await httpClient.AsCustomer(customerId).GetAsync(path);
+        var data = await response.ReadAsHydraResponseAsync<DefaultDeliveryChannel>();
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+        data.MediaType.Should().Be(mediaType);
+        data.Id.Should().Be($"{httpClient.BaseAddress}customers/{customerId}/spaces/{space}/defaultDeliveryChannels/{dbEntry.Entity.Id.ToString()}");
+    }
+    
+    [Fact]
+    public async Task Post_CreatesDefaultDeliveryChannelsSpaceNotAvailableInCustomer_200()
+    {
+        // Arrange
+        const string newCustomerJson = @"{
+  ""@type"": ""Customer"",
+  ""name"": ""api-test-customer-space"",
+  ""displayName"": ""My New Customer""
+}";
+        var customerContent = new StringContent(newCustomerJson, Encoding.UTF8, "application/json");
+        
+        var customerResponse = await httpClient.AsAdmin().PostAsync("/customers", customerContent);
+        var customerData = await customerResponse.ReadAsHydraResponseAsync<Customer>();
+        var customerId = int.Parse(customerData.Id!.Split('/').Last());
+        const int space = 5;
+        var path = $"customers/{customerId}/spaces/{space}/defaultDeliveryChannels";
+
+        string newDefaultDeliveryChannelJson = JsonConvert.SerializeObject(new DefaultDeliveryChannel()
+        {
+            MediaType = "image/tiff",
+            Policy = "default",
+            Channel = "iiif-img"
+        });
+        
+        // Act
+        var content = new StringContent(newDefaultDeliveryChannelJson, Encoding.UTF8, "application/json");
+        var response = await httpClient.AsCustomer(customerId).PostAsync(path, content);
+
+        // Assert
+        var data = await response.ReadAsHydraResponseAsync<DefaultDeliveryChannel>();
+
+        var dbEntry =
+            dlcsContext.DefaultDeliveryChannels.Include(d => d.DeliveryChannelPolicy)
+                .Single(d => d.Customer == customerId &&
+                             d.MediaType == "image/tiff" &&
+                             d.DeliveryChannelPolicy.Channel == "iiif-img" &&
+                             d.Space == space);
+
+        response.StatusCode.Should().Be(HttpStatusCode.Created);
+        data.MediaType.Should().Be("image/tiff");
+        data.Id.Should().Be($"{httpClient.BaseAddress}customers/{customerId}/spaces/{space}/defaultDeliveryChannels/{dbEntry.Id.ToString()}");
```

Comment:
Is `ToString()` needed here? I think the GUID would be converted implicitly with `{dbEntry.Id}`